src/components/atoms/CardHeader/CardHeader.tsx

```diff
@@ -0,0 +1,41 @@
+import { styled } from '@mui/material/styles';
+import { ReactNode } from 'react';
+import { Box, Typography } from 'src/components';
+
+export type CardHeaderProps = {
+  title: string;
+  description: string;
+  actions?: ReactNode;
+  children?: ReactNode;
+};
+
+const HeaderStyled = styled(Box)(({ theme: { palette } }) => ({
+  borderRadius: '4px 4px 0px 0px',
+  backgroundColor: palette.uiWhite[500],
+  display: 'flex',
+  alignItems: 'flex-start',
+  justifyContent: 'space-between',
+  padding: '16px 24px',
+  borderBottom: `1px solid ${palette.uiCoolGray[200]}`,
+  minHeight: '48px',
+}));
+
+export const CardHeader = ({
+  title,
+  description,
+  actions,
+  children,
+}: CardHeaderProps) => {
+  return (
+    <HeaderStyled>
+      <Box>
+        <Typography variant="h2">{title}</Typography>
+        <Typography variant="b2" color="textSecondary">
+          {description}
+        </Typography>
+      </Box>
+      {actions && <Box>{actions} </Box>}
```

Comment:
I'll just leave actions. I don't think we'll need something in the middle of the card header anytime soon. And if we do, the dev can arrange the space provided through styling. I don't think we need the children prop.

Reply:
+1 no need to design for something we don't have atm. We might need a different format or structure in the future